<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="m_lwRemainingFolders"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="m_pbSkip">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_lblImg">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>No image yet</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="m_laySave">
      <item row="1" column="1">
       <widget class="QPushButton" name="m_pbSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_leTextImg"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_acInputFolders"/>
    <addaction name="m_acOutputFolder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_acInputFolders">
   <property name="text">
    <string>Select input folders</string>
   </property>
  </action>
  <action name="m_acOutputFolder">
   <property name="text">
    <string>Select output folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbSkip</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SkipCurrentImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pbSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>637</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SkipCurrentImage()</slot>
  <slot>Save()</slot>
 </slots>
</ui>
